pkg/app/sensor/app.go

```diff
@@ -66,17 +66,17 @@ func getCurrentPaths(root string) (map[string]interface{}, error) {
 	return pathMap, nil
 }
 
-func startMonitor(errorCh chan error,
+func startMonitor(
+	errorChan chan error,
 	startAckChan chan bool,
-	stopWork chan bool,
-	stopWorkAck chan bool,
-	pids chan []int,
```

Comment:
pids is a placeholder for more targeted monitoring (will revisit this again later)

Reply:
Should I put the `pids` arg back?

Reply:
no need for now